members/forms.py

```diff
@@ -247,3 +247,11 @@ class Meta:
 class FunctionaryFilterForm(forms.Form):
     year = forms.IntegerField(required=False)
     functionary_role = forms.CharField(required=False)
+
+
+class MemberEditForm(forms.ModelForm):
+    email = forms.EmailField(required=True)
+
+    class Meta:
+        model = Member
+        fields = ['email', 'first_name', 'last_name', 'phone', 'address', 'zip_code', 'city', 'country']
```

Comment:
The form allows the user to set their first- and last name to blank, this should be fixed in the model by setting blank to false for it.